test/acceptance/Dockerfile.allure

```diff
@@ -0,0 +1,5 @@
+FROM yhxue911/python-allure
```

Comment:
we need to copy this image to quay due to the recent pull limits introduced by DockerHub

Reply:
this is not interactive session, so `-it` option can be removed.

Reply:
good points, will update mirror the image and update the command